src/main/java/kostka/moviecatalog/controller/UserController.java

```diff
@@ -45,4 +46,11 @@ public String getUserHomePage(final Model model) {
         model.addAttribute(LATEST_COMMENTS_KEY, externalCommentService.getLatest5Comments());
         return "index";
     }
+
+    @GetMapping("allMovies")
+    public String getAllMoviesPage(final Model model) {
+        LOGGER.info("rendering all movies page");
+        model.addAttribute(ALL_MOVIES_KEY, dbMovieService.getMoviesFromCacheWithKey(ALL_MOVIES_KEY));
```

Comment:
I have not realized that before. But is it OK to use RabbitMQ constants in the Thymeleaf templates?

I don't mind here actually. But maybe it will need some separation later.